### ISSUE TITLE
Symphony Fatal Database Error: Data too long for column 'handle-en' at row 1 (Output handles checkbox = false)

### ISSUE THREAD

**Issue by wdebusschere** [DeuxHuitHuit/multilingual_field#84]
Symphony 2.7.7
textboxfield 2.7.3 (and 2.8.1)
multilingual_field
Multilingual field description, adding 340 chars: Symphony Fatal Database Error: Data too long for column 'handle-en' at row 1

Text:
At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. 

Symphony Fatal Database Error: Data too long for column 'handle-en' at row 1
An error occurred while attempting to execute the following query

```
INSERT INTO `sym_entries_data_588` (`entry_id`, `handle-en`, `value-en`, `value_formatted-en`, `word_count-en`, `handle`, `value`, `value_formatted`, `word_count`, `handle-pt`, `value-pt`, `value_formatted-pt`, `word_count-pt`) VALUES ('71474', 'at-vero-eos-et-accusamus-et-iusto-odio-dignissimos-ducimus-qui-blanditiis-praesentium-voluptatum-deleniti-atque-corrupti-quos-dolores-et-quas-molestias-excepturi-sint-occaecati-cupiditate-non-provident-similique-sunt-in-culpa-qui-officia-deserunt-mollitia-animi-id-est-laborum-et-dolorum-fuga-et-harum-quidem-rerum-facilis-est-et-expedita-distinctio', 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. ', 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio.', '52', 'at-vero-eos-et-accusamus-et-iusto-odio-dignissimos-ducimus-qui-blanditiis-praesentium-voluptatum-deleniti-atque-corrupti-quos-dolores-et-quas-molestias-excepturi-sint-occaecati-cupiditate-non-provident-similique-sunt-in-culpa-qui-officia-deserunt-mollitia-animi-id-est-laborum-et-dolorum-fuga-et-harum-quidem-rerum-facilis-est-et-expedita-distinctio', 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. ', 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio.', '52', null, null, null, '0')
```

Why is there a handle-en if the output handles checkbox has been unchecked..

**Comment by wdebusschere** [comment]
SELECT * FROM `sym_fields_multilingual_textbox`
the text_handle for this field 588 = no, this is correct.

**Comment by wdebusschere** [comment]
Solved by updating to multilingual_field 3.5.18

**Comment by wdebusschere** [comment]
Other symphony user also got this, solved by updating the extension :)

> Greetings everyone! I updated a multilingual page (the relevant extensions: “Field: Multilingual Text Box”, “FLang detection gTLDs”, “Frontend Localisation”, “Languages”, “Page Lhandles”; Symphony and all extensions up-to-date) with Arabic (till then only left-to-right languages: German, French, Spanish, and English). This lead to an Symphony Fatal Database Error after filling the Arabic part of the text box and trying to save: “Data too long for column 'handle-ar' at row 1”. Going back and only keeping a few Arabic words within the field however worked out: saved successfuly. Guessing it may be due to the particular charset range I then tested with the full text from the German tab which interestingly enough also lead to the same Fatal Database Error. Only keeping a few German words in the Arabic field worked again. Anyone encountered such a behaviour before?
